.github/workflows/windows.yml

```diff
@@ -136,7 +136,7 @@ jobs:
       shell: cmd
       run: |
         call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
-        nmake test VERBOSE_FAILURE=yes HARNESS_JOBS=4
+        nmake test TESTS="-test_handshake-memfail" VERBOSE_FAILURE=yes HARNESS_JOBS=4
```

Comment:
Same as above

Reply:
same reasoning as above.  Its also much slower on windows for some reason (cause as of yet unknown).  But regardless I don't think we should run the memory failure test on every PR as it would be quite time consuming.